### PR TITLE
[IntelliJ Plugin] Update plugin description and release notes

### LINKED ISSUE #19313: Need to update the description of the plugin in IntelliJ plugin marketplace
Need to update the text in the plugin's page to align with the current messaging. 
Screenshot from 2019-10-05 23-16-13
The first sentence in particular, where it refers to Ballerina as a simple programming language.

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.

This PR also fixes #19313.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tool-plugins/intellij/src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+      in IntelliJ IDEA.
+      <br><br>For instructions on how to download, install, and use the features of the plugin, refer the
+      <a href="https://v1-0.ballerina.io/learn/tools-ides/intellij-plugin/">Ballerina IntelliJ plugin Guide</a>.
     ]]></description>
 
     <change-notes><![CDATA[
       <ul>
-        <li> Added diagnostics related plugin performance fix. </li>
-        <li> Fixed line-based code actions. </li>
+        <li> Added fixes for ballerina external library tracker.</li>
```
Comment (praneesha) [inline tool-plugins/intellij/src/main/resources/META-INF/plugin.xml]:
```suggestion
        <li> Added fixes for Ballerina external library tracker.</li>
```

File: tool-plugins/intellij/src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+        <li> Added debugger related minor fixes and improvements. </li>
```
Comment (praneesha) [inline tool-plugins/intellij/src/main/resources/META-INF/plugin.xml]:
```suggestion
        <li> Added debugger-related minor fixes and improvements. </li>
```

File: tool-plugins/intellij/src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/resources/META-INF/plugin.xml]: Done

File: tool-plugins/intellij/src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/resources/META-INF/plugin.xml]: Done

**Pull request merged**